packages/moonstone/internal/$L/$L.js

```diff
@@ -1,58 +1,26 @@
-/* global ILIB_MOONSTONE_PATH */
+import {getResBundle} from '../i18n/resBundle';
+import {getIStringFromBundle} from '@enact/i18n/src/resBundle';
 
-import ResBundle from '@enact/i18n/ilib/lib/ResBundle';
-import Locale from '@enact/i18n/ilib/lib/Locale';
-
-// The ilib.ResBundle for the active locale used by $L
-let resBundle;
+function toIString (str) {
+	const rb = getResBundle();
 
-/**
- * Creates a new ilib.ResBundle for string translation
- *
- * @returns {ilib.ResBundle} New ilib.ResBundle
- */
-function getResBundle () {
-	let currLoc = new Locale();
-	if (typeof ILIB_MOONSTONE_PATH === 'string' && (!resBundle ||
-			currLoc.getSpec() !== resBundle.getLocale().getSpec())) {
-		resBundle = new ResBundle({
-			loadParams: {
-				root: ILIB_MOONSTONE_PATH
-			}
-		});
-	}
-	return resBundle;
+	return getIStringFromBundle(str, rb);
 }
 
 /**
- * Deletes the current bundle object of strings and clears the cache.
- * @returns {undefined}
+ * Maps a string or key/value object to a translated string for the current locale.
+ *
+ * @function
+ * @memberof i18n/$L
+ * @param  {String|Object} str Source string
+ *
+ * @returns {String} The translated string
  */
-function clearResBundle () {
-	delete ResBundle.strings;
-	delete ResBundle.sysres;
-	resBundle = null;
-}
-
-
-/**
-* Localized internal Moonstone strings from iLib translations.
-*
-* @param {String} str - String to be localized.
-* @returns {String} Localized string.
-*/
 function $L (str) {
-	const rb = getResBundle();
-	const isObject = typeof str === 'object';
-	if (!rb) {
-		return isObject ? str.value : str;
-	}
-	return isObject ? rb.getString(str.value, str.key).toString() : rb.getString(str).toString();
+	return String(toIString(str));
 }
 
 export default $L;
 export {
-	$L,
-	getResBundle,
-	clearResBundle
```

Comment:
Just one note that moving/renaming `clearResBundle` from the module will require a PR to `@enact/dev-utils`/`cli` too (see https://github.com/enactjs/dev-utils/blob/master/plugins/SnapshotPlugin/snapshot-helper.js#L41)

Reply:
Added `clearResBundle` in `internal/$L`. FYI: 
```	delete ResBundle.strings;
	delete ResBundle.sysres;```
seems unnecessary, but added to keep the parity with how `i18n/src/resBundle.clearResBundle` does clearing.